src/Views/AppInfoView.vala

```diff
@@ -743,6 +751,23 @@ namespace AppCenter.Views {
 
             var size = yield package.get_download_size_including_deps ();
             size_label.update (size, package.is_flatpak);
+
+            if (package.deprecated_dependencies_number > 0 && !is_eol_shown) {
+                is_eol_shown = true;
+                var end_of_life = new ContentType (
+                    _("End of Life"),
```

Comment:
Ironically, even though I'm aligning on this terminology in code, I think it won't mean much to users. We should probably do something more straightforward, like:

```suggestion
                    _("Outdated"),
```